ims/einstein/ceph.py

```diff
@@ -117,6 +129,22 @@ def clone(self, parent_img_name, parent_snap_name, clone_img_name):
             raise file_system_exceptions.ArgumentsOutOfRangeException()
 
     @log
+    def list_children(self, img_id, parent_snap):
+        """
```

Comment:
This is used by tests [here](https://github.com/CCI-MOC/ims/pull/101/files#diff-3c078ee34b92c8428b97dd0f842eacd2R174). I think it should stay here because of the following reasons:
* all methods talking to ceph or any file system should be grouped together (that is the driver). I have added a docstring that this method is only needed by tests.
* If we support another file system, then this method (an abstract version of course) can be a part of the file system interface because we would require it to be implemented for our tests.